### PR TITLE
Moved the `strip_accents` & `strip_accents_and_lower` functions

### PR DESCRIPTION
**Pull request by BaptisteB-A**
Moved the `strip_accents` & `strip_accents_and_lower` functions from `source/autocomplete/autocomplete.h`
[Jira ticket NAVP-1446](https://jira.kisio.org/browse/NAVP-1446)

### CODE REVIEW

File: functions.cpp [hunk 1]
```diff
@@ -96,4 +96,22 @@ std::string absolute_path(){
     }
 }
 
+std::string strip_accents(std::string str) {
```
Comment (nonifier) [inline functions.cpp]: Where there any tests around this function that need to be moved to utils as well?

File: functions.h [hunk 2]
```diff
@@ -117,13 +119,20 @@ unique_ptr<T> make_unique(Args&& ...args) {
...
+ * pseudo natural sort:
+ * if both string carry integer, we compare them, else we compare the string
  */
 struct pseudo_natural_sort {
     bool operator() (const std::string&, const std::string&) const;
 };
 
+/**
+ * strip_accents_and_lower:
+ * We compare strings: integers and accents are taken in account for the sort
```
Comment (nonifier) [inline functions.h]: This comment doesn't seem relevant :confused: 
You probably want to update or remove it.

File: functions.cpp (hunk 1 above)
Comment (BaptisteB-A) [reply functions.cpp]: Thanks, done [here](https://github.com/CanalTP/utils/pull/85/commits/82d891c05c447ea8f017858b13f379f9a1b7b2c3) !

File: functions.h (hunk 2 above)
Comment (BaptisteB-A) [reply functions.h]: Alright, I removed it

**Pull request merged**